src/content/en/tools/chrome-devtools/javascript/add-breakpoints.md

```diff
@@ -123,25 +130,95 @@ breakpoints** button
-
-![DOM change breakpoint][dcb]
-
-**Tip**: These are not exclusive options. You can have two or more of the
-options enabled on any given node.
-
-Here's more information about each type of breakpoint:
+There's a bug somewhere in your code that's incorrectly changing, deleting,
+or adding a DOM node. DevTools provides a fast way to root cause this issue:
+DOM change breakpoints.
+
+Rather than manually searching around for the code that's causing the change,
+DevTools enables you to set a breakpoint on the node. Whenever the node, or
+in some cases one of its children, is added, deleted, or changed,
+DevTools pauses the page and takes you to the exact line of code that's
+causing it.
+
+Below is a live demo for learning how to set DOM change breakpoints.
+Clicking on **Increment** increments **Count** by one. Try it now.
+
+Your goal in this interactive tutorial is to set a DOM change breakpoint
+that is triggered when **Count** increases, so that you can inspect the
+code that is modifying **Count**.
+
+{% framebox height="auto" %}
+<p><b>DOM Change Breakpoints Demo</b></p>
+<button>Increment</button>
+<p>Count: <span>0</span></p>
+<script>
+var buttons = document.querySelectorAll('button');
+var increment = buttons[0];
+var toggle = buttons[1];
+var count = document.querySelector('span');
+increment.addEventListener('click', function() {
+  count.textContent = parseInt(count.textContent) + 1;
+});
+</script>
+{% endframebox %}
+
+To **add the DOM change breakpoint**:
+
+1. Right-click on **Count** and select **Inspect**. DevTools highlights
+   the node blue. It should be a `<p>` node. You can verify that you're on
+   the right node by double-clicking it, which expands the node so that you
+   can see its contents.
+
+1. Right-click on the highlighted node and select **Break on** >
+   **Subtree Modifications**. The blue icon ![DOM breakpoint 
+   icon][icon]{:.inline} to the left of the node indicates that a DOM
+   breakpoint is set on the node. It's a little hard to see the icon while
+   the node is highlighted, since it's a blue icon against a blue
+   background.
+
+1. Back on the demo, click **Increment**. DevTools pauses the page, goes
+   to **Sources**, and highlights the line of code in the script that is
+   causing the change.
+
+1. Press **Resume script execution** ![resume script execution
+   button](imgs/resume-script-execution.png){:.inline} twice to resume
+   script execution. You need to press it twice because the breakpoint is
+   triggered once when the count text is deleted, and then again when the
+   text is updated with the new count.
+
+To break when an attribute of the selected node is changed, or when the 
+selected node is deleted, just select **Attributes modifications** or
+**Node Removal** instead of **Subtree Modifications** in step 2 above.
+
+Tip: These breakpoints are not exclusive. You can have two or all three of these breakpoints enabled on a single node at the same time.
```

Comment:
I tried note, but it's not really jiving. I'll just leave it plaintext and maybe advocate for a new tip style.